### ISSUE TITLE
How can I get the autoIncrements key with the plain SQL?

### ISSUE THREAD

**Issue by amoszhou** [slick/slick#965]
Now, I  use  the StaticQuery.u + "insert ......"  to  insert data into db.
   But my question is  How i can get the autoIncrements key value ?  for now the slick seems not support.

**Comment by szeiger** [comment]
You'll have to use something like "insert ... returning ..." in SQL. The JDBC abstraction for returning generated keys is not supported.

**Comment by cvogt** [comment]
would this allow access to the generated keys function https://github.com/slick/slick/issues/1038 ?

**Comment by szeiger** [comment]
No, you could use #1038 to request keys but you wouldn't be able to read the keys back.
